### ISSUE TITLE
[BUG] Multi-Project Issue: Display and Execution of Triggers in Other Projects After Transfer

### ISSUE THREAD

**Issue by mrykin** [mage-ai/mage-ai#4836]
### Mage version

0.9.67

### Describe the bug

1. Why when I choose first project, I can see triggers from another project in multi project platform? Is it normal?
v0.9.67
![image](https://github.com/mage-ai/mage-ai/assets/13833432/a9bf5261-1586-4575-a999-70b994c2dd8a)
2. When I start main (multi project) - my triggers was duplicate
![image](https://github.com/mage-ai/mage-ai/assets/13833432/d682e00d-095c-4b7a-85b0-7d3c1f46cc6c)
3. I start mage with use old DB, now, I see new name and old name in projects
![image](https://github.com/mage-ai/mage-ai/assets/13833432/9d4b0c50-9e2a-4f81-a93c-37991df9618b)
4. On the overview I see stats from mage_ct (DB from migrating project)
![image](https://github.com/mage-ai/mage-ai/assets/13833432/a44232ca-9d0e-4665-a3a3-6bf14d75ad10)

File with settings.yaml
[settings.yaml.zip](https://github.com/mage-ai/mage-ai/files/14763878/settings.yaml.zip)

### To reproduce

1. Create parent dir
2. Transfer old project inside parents dir
3. Create settings.yaml
4. I didn't transfer defaults folders inside others project (ornp, clients)
6. Start docker like command
sudo docker run -d -p 6789:6789 \
  -v $(pwd):/home/src \
  --name mage_ct \
  --restart=always \
  -e REQUIRE_USER_AUTHENTICATION=1 \
  -e REQUIRE_USER_PERMISSIONS=1 \
  -e MAGE_PUBLIC_HOST=MY_URL:6789 \
  mageai/mageai:latest /app/run_app.sh mage start main

### Operating system

Mac os, Chrome

**Comment by johnson-mage** [comment]
@mrykin Yes, we can reproduce the issues you mentioned and are addressing it today.

**Comment by johnson-mage** [comment]
@mrykin 
> 1. Why when I choose first project, I can see triggers from another project in multi project platform? Is it normal?
Currently, the triggers from another project are visible, but it should only show the triggers from your currently selected project. We'll fix this.

> 2. When I start main (multi project) - my triggers was duplicate
Were the triggers that were duplicated all saved as "triggers in code"? Meaning those triggers' pipelines had their triggers saved in a `triggers.yaml` file? It seems like new triggers are getting created from the trigger config files (even though they already exist in the db) due to the change in `repo_paths` after migrating to multi-project platform. Will see if we can avoid this duplicate trigger issue.

> 3. I start mage with use old DB, now, I see new name and old name in projects
Can you clarify what you mean here? What was the expected behavior? 

> 4. On the overview I see stats from mage_ct (DB from migrating project)
Was the previous project (that you migrated from) a single-project setup? Do you see the pipeline runs (e.g. in the selected project's Pipeline Runs page) from the previous project after migrating to the multi-project platform? What is the expected behavior here (e.g. do you expect to see the pipeline runs from the previous project, as well as the stats for them, to appear in the Overview page?

Feel free to DM me in our Slack channel if you'd like, may be faster to communicate that way. If you're able to start with a fresh slate for your metadata db (meaning no existing triggers or pipeline runs in the db) when migrating to the multi-project platform, that would be ideal. If you have triggers saved in code, the triggers.yaml files would automatically be used to create new triggers in the db. Nonetheless, thanks for bringing up these issues. We'll work on making the migration from an existing project to the multi-project platform smoother.

**Comment by mrykin** [comment]
> Currently, the triggers from another project are visible (intentionally), but we will think about how to improve the UX since users might expect it to only show the triggers from the currently selected project.
I don't understand the logic of this. It's not convenient from the point of view that I see other people's triggers (if I assume a separate project as independent). And I thought that this was precisely the problem causing my pipelines to run incorrectly.

> Were the triggers that were duplicated all saved as "triggers in code"? Meaning those triggers' pipelines had their triggers saved in a triggers.yaml file? It seems like new triggers are getting created from the trigger config files (even though they already exist in the db) due to the change in repo_paths after migrating to multi-project platform. Will see if we can avoid this duplicate trigger issue.

Yes, this setting was activated. I use it as an additional backup of settings, so that everything remains in the code if necessary to transfer."

> Can you clarify what you mean here? What was the expected behavior?
My expectations were: I had one project that had an old name, I start a new project using data from the old one. I expect to see either the old name (if possible) or the new one, but not both at the same time. Now I see two project names, which are actually one: mage_ct - parent of the data, marketing - successor.

> Was the previous project (that you migrated from) a single-project setup? Do you see the pipeline runs
Yes, I migrated from single-project. What I mean. In each project, I want to see statistics for that project only, and the breadcrumbs say exactly that I am in the overview of the child project, not of the entire general."

**Comment by johnson-mage** [comment]
@mrykin Thanks for the responses. There are definitely improvements that can be made with the multi-project platform UX in general (e.g. confining triggers to just the currently selected project). There were certain engineering considerations made for extensibility, which resulted in data like other projects' triggers being visible in the current project, but this is all great feedback we'll take into account when making updates.

**Comment by wangxiaoyou1993** [comment]
We are no longer continuing support for multi-project mode in the OSS version, so we’re closing this ticket.

For multi-project support, please refer to Workspace Mode in Mage Pro:
https://docs.mage.ai/guides/developer-ux/workspaces#mage-pro-workspaces
